server/test/article.spec.js

```diff
@@ -0,0 +1,98 @@
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../index');
+const pool = require('../models/db')
+
+
+const title = "Political Education"
+
+// ASSERTION STYLE
+chai.should();
+
+chai.use(chaiHttp);
+ let token;
+
+describe('POST ARTICLE ',()=>{
+    // LOGIN FIRST IN ORDER TO GET YOUR VALID TOKEN
+    describe('POST /auth/v1/login',()=>{
```

Comment:
why have each test in a different describe?. login a user in should be inside a before hook